src/device-registry/models/KnowYourAirTask.js

```
@@ -113,11 +113,13 @@
     try {
       const inclusionProjection = constants.KYA_TASKS_INCLUSION_PROJECTION;
       const exclusionProjection = constants.KYA_TASKS_EXCLUSION_PROJECTION(
-        filter.category ? filter.category : "none"
+        filter.path ? filter.path : "none"
       );
-      if (!isEmpty(filter.category)) {
-        delete filter.category;
+
+      if (!isEmpty(filter.path)) {
+        delete filter.path;
       }
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)